.github/labeler.yml

```
@@ -0,0 +1,48 @@
+# Made with love
+# see https://github.com/actions/labeler
+# for more informations
+
+# Add 'assets' label to any assets file changes within the source dir
+Assets:
+- app/src/main/assets/**/*
+
+# Add 'Github' label to any git file changes within the source dir
+Github:
+- .github/**/*
+
+# Add 'Core' label to any change within the 'core' package
+Core:
+- app/src/main/java/com/trianguloy/urlchecker/**/*
```

```suggestion
- app/src/main/java/**/*
- app/src/main/kotlin/**/*
```

although it would be strange to add files outside the main package, they are indeed possible. the 'java' (and 'kotlin', currently unused but maybe in the future it is) folders are where all the 'core' code must be added

---

Thanks for the clarifications, and the corrections, they have been added